networkx/algorithms/regular.py

```diff
@@ -78,6 +90,13 @@ def k_factor(G, k, matching_weight="weight"):
     G2 : NetworkX graph
         A k-factor of G
 
+    Examples
+    --------
+    >>> G = nx.Graph([(1, 2), (2, 3), (3, 4), (4, 1)])
+    >>> G2 = nx.k_factor(G, k=1)
+    >>> G2.edges()
+    EdgeView([(1, 2), (3, 4)]
```

Comment:
There is a missing `)`

```suggestion
    EdgeView([(1, 2), (3, 4)])
```